inc/notificationtarget.class.php

```diff
@@ -1536,6 +1536,21 @@ function getUserByField($field, $search_in_object = false) {
    }
 
    /**
+    * Add new recipient with lang to current recipients array
+    *
+    * @param array $data Data (users_id, lang[, field used for notification])
+    *
+    * @deprecated Use NotificationTarget::addToRecipientsList()
+    *
```

Comment:
Yes you're right; thank you!
I forget to set the version on all deprecateds added in notifications rework; will fix this.